ask-sdk-controls/ide/vscode/launch.json

```diff
@@ -7,21 +7,19 @@
         {
             "type": "node",
             "request": "launch",
-            "name": "Run tests (LOG=info)",
-            //"preLaunchTask": // use $npm run watch for continuous compilation
+            "name": "Run tests (INFO)",
             "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
             "args": [
-                "--require",
-                "source-map-support/register",
                 "--timeout",
                 "999999",
                 "--colors",
-                "\"${workspaceFolder}/dist/test/**/*.spec.js\""
+                "-r",
+                "ts-node/register",
+                "${workspaceFolder}/test/**/*.spec.ts"
```

Comment:
I think we need to keep \" "\ otherwise not all test files will be executed. Could you please check whether all files are executed without \" "\ ?

Reply:
The extra quotes appears to be necessary on macOS but not Ubuntu.  I couldn't find any information on how to make this platform agnostic so I've addressed it by adding a comment.

Reply:
@mikelid Did you try using `cross-env` cmd, which is used in package.json `npm run test` config on Ubuntu with quotes?

Reply:
I tried using cross-env as follows, but it was not successful as the final path expansion appears to be done by the shell before any fixups by cross-env.  

```
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Run tests (INFO)",
            "program": "${workspaceFolder}/node_modules/cross-env/src/bin/cross-env",
            "args": [
                "TS_NODE_FILES=true",
                "${workspaceFolder}/node_modules/mocha/bin/_mocha",
                "--timeout",
                "999999",
                "--colors",
                "-r",
                "ts-node/register",
                "\"${workspaceFolder}/test/**/*.spec.ts\"" 
            ],
            "env": {
                //"DEBUG": "error:*, warn:*, info:*"  // uncomment for logs
            },
            "smartStep": true,
            "skipFiles": [
                "node_modules/**",
                "<node_internals>/**/*.js",
                "node_internals/**"
            ],
            "console": "integratedTerminal"
        }
    ]
}
```

Error:
```zsh: no matches found: "/home/mliddell/workplace/github/alexa-skills-kit-sdk-for-nodejs/ask-sdk-controls/test/**/*.spec.ts"
```

Reply:
Thanks for confirming @mikelid, Then I guess your changes are fine 👍